include/openssl/evp.h

```diff
@@ -246,63 +246,6 @@ int (*EVP_MD_meth_get_ctrl(const EVP_MD *md))(EVP_MD_CTX *ctx, int cmd,
 #define EVP_MD_CTX_FLAG_FINALISE 0x0200
 /* NOTE: 0x0400 and 0x0800 are reserved for internal usage */
 
-#ifndef OPENSSL_NO_DEPRECATED_3_0
-OSSL_DEPRECATEDIN_3_0
-EVP_CIPHER *EVP_CIPHER_meth_new(int cipher_type, int block_size, int key_len);
-OSSL_DEPRECATEDIN_3_0
-EVP_CIPHER *EVP_CIPHER_meth_dup(const EVP_CIPHER *cipher);
-OSSL_DEPRECATEDIN_3_0
-void EVP_CIPHER_meth_free(EVP_CIPHER *cipher);
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_iv_length(EVP_CIPHER *cipher, int iv_len);
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_flags(EVP_CIPHER *cipher, unsigned long flags);
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_impl_ctx_size(EVP_CIPHER *cipher, int ctx_size);
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_init(EVP_CIPHER *cipher,
-    int (*init)(EVP_CIPHER_CTX *ctx,
-        const unsigned char *key,
-        const unsigned char *iv,
-        int enc));
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_do_cipher(EVP_CIPHER *cipher,
-    int (*do_cipher)(EVP_CIPHER_CTX *ctx,
-        unsigned char *out,
-        const unsigned char *in,
-        size_t inl));
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_cleanup(EVP_CIPHER *cipher,
-    int (*cleanup)(EVP_CIPHER_CTX *));
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_set_asn1_params(EVP_CIPHER *cipher,
-    int (*set_asn1_parameters)(EVP_CIPHER_CTX *,
-        ASN1_TYPE *));
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_get_asn1_params(EVP_CIPHER *cipher,
-    int (*get_asn1_parameters)(EVP_CIPHER_CTX *,
-        ASN1_TYPE *));
-OSSL_DEPRECATEDIN_3_0
-int EVP_CIPHER_meth_set_ctrl(EVP_CIPHER *cipher,
-    int (*ctrl)(EVP_CIPHER_CTX *, int type,
-        int arg, void *ptr));
-OSSL_DEPRECATEDIN_3_0 int (*EVP_CIPHER_meth_get_init(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *ctx,
-    const unsigned char *key,
-    const unsigned char *iv,
-    int enc);
-OSSL_DEPRECATEDIN_3_0 int (*EVP_CIPHER_meth_get_do_cipher(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *ctx,
-    unsigned char *out,
-    const unsigned char *in,
-    size_t inl);
-OSSL_DEPRECATEDIN_3_0 int (*EVP_CIPHER_meth_get_cleanup(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *);
-OSSL_DEPRECATEDIN_3_0 int (*EVP_CIPHER_meth_get_set_asn1_params(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *,
-    ASN1_TYPE *);
-OSSL_DEPRECATEDIN_3_0 int (*EVP_CIPHER_meth_get_get_asn1_params(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *,
-    ASN1_TYPE *);
-OSSL_DEPRECATEDIN_3_0 int (*EVP_CIPHER_meth_get_ctrl(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *, int type,
-    int arg, void *ptr);
-#endif
-
```

Comment:
What should we do for the unused EVP_CIPH values below? I think at least they should get a comment above them that they are unused and might be deprecated in future?